corehq/toggles.py

```diff
@@ -860,8 +860,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 CUSTOM_INSTANCES = StaticToggle(
     'custom_instances',
-    'Enikshay: Inject custom instance declarations',
+    'Inject custom instance declarations',
     TAG_CUSTOM,
+    description=(
+        'Enables the insertion of custom instances into a case list configuration. '
+        'Currently used by SimPrints-integrated projects.'
+    ),
     namespaces=[NAMESPACE_USER, NAMESPACE_DOMAIN],
```

Comment:
Do any users have this enabled? I think we'll only want to have this enabled on the domain to make sure that app edits don't wipe this value? (I realize that this was pre-existing, but it does seem bad)

Reply:
Good call. No users on `www` or `india`- think that covers it?

Reply:
Yeah. I think that should cover it